[BE] Email that someone has applied to the job

nspc97 opened the pull request:
[BE] Email that someone has applied to the job
https://trello.com/c/vbtdfGfE/66-beedit-error-response-code

Commit:
[BE]Edit error response code
https://trello.com/c/vbtdfGfE/66-beedit-error-response-code

Commit:
[BE] Email that someone has applied to the job
https://trello.com/c/hGpAE9tV/54-be-email-that-someone-has-applied-to-the-job

SpartakusMd commented on src/Controller/ActivityController.php:
```diff
@@ -538,6 +547,22 @@ public function applyForActivity(Activity $activity, ActivityUserRepository $act
         if ($activityUser !== null) {
             return new JsonResponse(['message' => 'You cannot apply!'], Response::HTTP_BAD_REQUEST);
         }
+        $message = (new Swift_Message('Info Pentabee'))
```
Use `Someone applied your job: <job_name>`

Commit:
[BE] Email that someone has applied to the job
https://trello.com/c/hGpAE9tV/54-be-email-that-someone-has-applied-to-the-job

Responsive html email
